app/data.json

```diff
@@ -62,5 +62,8 @@
         "Python"
       ]
     }
-  ]
+  ],
+  
+  "pages": [{"name": "Home", "url": "http://127.0.0.1:5000/"}, {"name": "Map", "url": "http://127.0.0.1:5000/map"}, {"name": "About Us", "url": "http://127.0.0.1:5000/aboutme"}]
```

Comment:
I first intended to use this file to "mirror" a database that fetches information related to our users. The "configuration" parameter you're suggesting would fit better in a file like: `app/config.json` (which is something I'm totally down for).

Else, I think hard-coding these values inside the HTML sounds like a straightforward solution as well.

Reply:
I'm open to both solutions.

Reply:
I'm interested in the ```app/config.json``` idea! I need some more clarification on what that would look like however.

Reply:
I've hardcoded the URLs values for now.